server/app/com/xsn/explorer/models/persisted/Transaction.scala

```diff
@@ -16,17 +16,25 @@ object Transaction {
       fromOutputIndex: Int,
       index: Int,
       value: BigDecimal,
-      addresses: List[Address]
+      addresses: List[Address],
+      script: Option[HexString]
```

Comment:
A problem is that while the addresses and the value from the spent output are stored on the input to avoid another query, the script isn't, which means that the only way you could get the script is by filling it by yourself, also, I don't see you are actually filling this value.

I understand this is the simplest way, I'll think about alternatives.

Reply:
yeah, this is for testing purposes only at the moment since the current query does not include the script.